cosmo_tester/framework/examples/on_manager.py

```diff
@@ -49,3 +53,22 @@ def check_files(self):
                 file_path = self.inputs['path'] + '_' + instance.id
                 content = self.manager.get_remote_file_content(file_path)
                 assert content == expected_content
+
+    def uninstall(self):
```

Comment:
Is it worth to add force flag and ignore error ?

Reply:
No need- this is on-manager only. I'll look more into that for the off-manager one once I reach that point.